src/pybind/mgr/orchestrator/module.py

```diff
@@ -349,6 +349,7 @@ def _add_host(self,
             labels = labels[0].split(',')
```

Comment:
This is minor but it might be nice to change the commit message to say "HostSpec validation" rather than "hostname validation" because that's closer to what it's really doing.

Reply:
Done :+1: